### ISSUE TITLE
Lazy load gifs

### ISSUE THREAD

**Issue by nouveller** [daviddarnes/currentstat.us#16]
This should stop jerky page load.

**Comment by daviddarnes** [comment]
Good call, would we use a package for this? Might mean we need to get a package manager involved.

**Comment by nouveller** [comment]
Yeah, the package manager thing, is a thing. As we're still hot loading Vue.

We obviously don't want to commit the assets in, but they have to be to be served on Github Pages. One method I've seen is to have `master` all nice and clean, package managers and all. Then, add a deploy script which compiles any external CSS/JS dependencies and commits them to the `gh-pages` branch. Not a bad option as we'd never commit directly to that branch anyway.

**Comment by daviddarnes** [comment]
Well personally I'm not bothered about committing the compiled packages. We'd only be adding in the `javascripts/` directory anyway, however I'm open to the implementation as long as it's smooth.

Could this be done with travisCI?
